src/App.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import { Route, Routes } from 'react-router-dom';
+import Header from './components/Header';
+import Books from './components/Books';
+import Category from './components/Category';
+
+function App() {
+  return (
+    <div className="bookstore">
+      <Header />
+      <div className="container">
+        <Routes>
+          <Route path="/" element={<Books />} />
+          <Route path="/catagory" element={<Category />} />
+        </Routes>
+      </div>
```

Comment:
Nice Job 👍 
![Capture](https://user-images.githubusercontent.com/63972670/200150976-6fd2bcc8-0ce3-40a2-9692-8ec703e2cda6.PNG)

Reply:
Thank you @OLIPLICHE for that review